test/integration/che/cheAPI.go

```diff
@@ -192,6 +192,7 @@ func doRequest(requestType, url, data string) ([]byte, int, error) {
 	if err != nil {
 		return nil, res.StatusCode, err
 	}
+	defer res.Body.Close()
```

Comment:
👍 , @agajdosi I would say as soon as the res created then only we should defer it. like in the L#186

Reply:
@praveenkumar req does not have Close() function, I can only set req.Close=true which would prevent from reusing the connection. It might help, WDYT?

From https://golang.org/pkg/net/http/#Request:
```
        // For client requests, setting this field prevents re-use of
        // TCP connections between requests to the same hosts, as if
        // Transport.DisableKeepAlives were set.
        Close bool
```

Reply:
I have updated the PR so the Close option is set to be enabled.